app/models/delegates/user_mvi_delegate.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+class UserMviDelegate
+  MVI_SERVICE = VetsAPI::Application.config.mvi_service
+
+  def initialize(user)
+    @user = user
+  end
+
+  def create
+    raise Common::Exceptions::ValidationErrors, @user unless @user.valid?
+    message = create_message
+    raise Common::Exceptions::ValidationErrors, message unless message.valid?
```

Comment:
Just an FYI - Vallidation Error of this nature are useful but I wonder if instead we should be doing some kind of NotAuthroized exception. This is going to render information to the client / user as a 422 vs 401